x/vesting/spec/07_clients.md

```diff
@@ -60,6 +60,17 @@ Allows users to create a transfer unvested amount out of a ClawbackVestingAccoun
 evmosd tx vesting clawback ADDRESS [flags]
 ```
 
+**`update-vesting-funder`**
+
+Allows users to update the funder of an existent ClawbackVestingAccount. Must be requested by the original funder address (--from). To perform this action, the user needs to provide two arguments:
+
+1. the new funder address
+2. the vesting account address
+
+```go
+evmosd tx vesting update-vesting-funder NEW_FUNDER_ADDRESS VESTING_ACCOUNT_ADDRESS [flags]
```

Comment:
Maybe one could explicitly add the `--from FUNDER_ADDRESS` here so it's clear that it is mandatory and if someone copy-pastes this, he/her will know immediately to add this